app/views/game_masters/show.html.erb

```diff
@@ -1,22 +1,79 @@
 <p id="notice"><%= notice %></p>
 
-<p>
-  <strong>Table:</strong>
-  <%= @game_master.table.session.name %> <%= @game_master.table.scenario.name %>
-</p>
+<!-- breadcrumbs -->
+<ol class="breadcrumb">
+  <li><%= link_to "#{@event.name}", event_path(@event) %></li>
+  <li><%= link_to "#{@session.long_name}", event_session_path(@event, @session) %></li>
+  <li><%= link_to "#{@table.long_name}", event_session_table_path(@event, @session, @table) %></li>
+  <li>Game Master Information</li>
+</ol>
 
-<p>
-  <strong>GM:</strong>
-  <%= @game_master.user_event.user.name %>
-</p>
 
+<div class="page-header">
+  <h1><%= @event.name %></h1>
+  <h2><%= @session.name %>
+    <small><%= @session.timeslot %></small>
+  </h2>
+  <h3><%= @table.long_name %></h3>
+</div>
 
-<p>
-  <strong>Table Number:</strong>
-  <%= @game_master.table_number %>
-</p>
+<% player = @game_master&.user_event&.user %>
+<%#
+table info
+table number
+session ?
+%>
+<h4>Table Information</h4>
+<div class="panel panel-padded" style="border: solid thin black;">
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Table Location:</strong></div>
+    <div class="col-md-6 col-sm-5"><%= @table.location %></div>
+  </div>
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Table Information:</strong></div>
+    <div class="col-md-6 col-sm-5">Put stuff here</div>
+  </div>
+</div>
 
+<h4>Game Master's Information</h4>
+<div class="panel panel-padded" style="border: solid thin black;">
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Game Master:</strong></div>
+    <div class="col-md-6 col-sm-5"><%= player.formal_name %></div>
+  </div>
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Email:</strong></div>
+    <div class="col-md-6 col-sm-5"><%= player.email %></div>
+  </div>
+</div>
 
 
+<h4>Online Game Information</h4>
+<div class="panel panel-padded" style="border: solid thin black;">
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Chat Server:</strong></div>
```

Comment:
What's the difference between a chat server and a chat room? Like the chat server is the overall discord, and the chat room is the specific channel for the game? If so, I'd call the other one 'Game Chat Room'

Reply:
@panpear yes. 
Chat server will be the con's chat server.
The room is the specific room for the game.